### PR TITLE
Remove winston for logging (not web compatible)

### PR DESCRIPTION
**Pull request by rchiodo**
Winston is nodejs only. 

This changes our logging to match what the Python extension did (much simpler).

It does however maintain the trace decorators that Don added.

### CODE REVIEW

**Comment by rchiodo** [comment]
Telemetry and localization also had to be web specific (used by logging) so moved those too.

Most of the changes are in localize.ts and the logging code. Everything else is just name changes.

File: src/platform/logging/types.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    Warn = 30,
+    Info = 20,
+    Debug = 10,
+    Trace = 5,
+    Off = 0
+}
+
+export type Arguments = unknown[];
+
+export interface ILogging {
```
Comment (amunger) [inline src/platform/logging/types.ts:19]: Seems like it should be called ILogger. Is this just what Python named it?

Comment (rchiodo) [reply src/platform/logging/types.ts:19]: Yeah I just copied it. I agree. I'll change the name.

File: src/extension.node.ts [hunk 2]
```diff
@@ -210,9 +214,28 @@ async function activateComponents(
     return activateLegacy(context, serviceManager, serviceContainer);
 }
 
+function addConsoleLogger() {
+    if (process.env.VSC_JUPYTER_FORCE_LOGGING) {
+        let label = undefined;
+        // In CI there's no need for the label.
+        const isCI = process.env.TF_BUILD !== undefined || process.env.GITHUB_ACTIONS === 'true';
```
Comment (IanMatthewHuff) [inline src/extension.node.ts:221]: Aren't we already importing an "isCI" into this file?

Comment (rchiodo) [reply src/extension.node.ts:221]: Thanks. Didn't notice that.

**Review by IanMatthewHuff — APPROVED** [review]

**Review by amunger — APPROVED** [review]

**Comment by rchiodo** [comment]
I love our tests 🥇 . Windows test failed for interrupt because the daemon path couldn't be constructed for the interrupt daemon module. Root cause - path-browserify won't join rooted paths. It just drops the path altogether.

**Pull request merged**